python/xorbits/_mars/dataframe/reduction/tests/test_reduction_execution.py

```diff
@@ -1192,3 +1192,32 @@ def test_gpu_multi_agg_methods(setup_gpu):
 
     res = mars_series.agg(["sum", "prod"]).execute().fetch(to_cpu=False)
     pd.testing.assert_series_equal(expected, res.to_pandas())
+
+
+# TODO: support cuda
+# @support_cuda
+def test_agg_on_same_funcs(setup_gpu, gpu):
```

Comment:
Nit: Better to add tests to test the following scenarios: ``.agg`` (check if it is equivalent to aggregate) ``.agg((g1, g1))`` (check two real same functions)

Reply:
Good point! Will add test cases for these scenarios.

Reply:
Good point! Will add test cases for these scenarios.